lambdas/generate-gro-xml/src/main/java/uk/gov/di/data/lep/dto/FieldOptions.java

```diff
@@ -0,0 +1,107 @@
+package uk.gov.di.data.lep.dto;
+
+import uk.gov.di.data.lep.library.dto.GroAddressStructure;
+import uk.gov.di.data.lep.library.dto.GroPersonNameStructure;
+import uk.gov.di.data.lep.library.dto.PersonBirthDateStructure;
+import uk.gov.di.data.lep.library.dto.PersonDeathDateStructure;
+import uk.gov.di.data.lep.library.enums.GenderAtRegistration;
+import uk.gov.di.data.lep.library.enums.GroVerificationLevel;
+
+import java.time.LocalDate;
+import java.util.List;
+
+public class FieldOptions {
+    public final List<String> title = List.of(
```

Comment:
Can I double check the source of these titles, are they our ones? I think we don't pass this on to DWP so maybe doesn't matter loads, but would be good to make sure we've got the complete list

Reply:
I just came up with some examples, it can be any string up to 128 chars